boards/beagle/beagleconnect_freedom/beagleconnect_freedom.dts

```diff
@@ -206,10 +206,6 @@
 	};
 };
 
-&ieee802154 {
-	status = "okay";
-};
-
```

Comment:
A lot of zephyr networking samples (zperf, http_server, etc) will complain if both 2.4g and subg ieee802154 are enabled simultaneously.
This leads to a lot of new people getting confused with the network code not working without any clear errors.
As you can see, we already set the alias `zephyr,ieee802154 = &ieee802154g;`, so I think it would be best to enable only subg by default.

Reply:
Okay. Perhaps add this as a separate commit with the above explanation?